### ISSUE TITLE
(team) You are signed out. Please run the 'team signin' command.

### ISSUE THREAD

**Issue by dmumladze** [microsoft/azure-repos-vscode#355]
I'm using VSCode 1.19.0 with latest vsts plugin. I have Visual Studio 2017 Pro and TFS connection is working Ok there, but VSCode seems to be complaining about it...

settings.json
```
"tfvc.location": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional\\Common7\\IDE\\CommonExtensions\\Microsoft\\TeamFoundation\\Team Explorer\\TF.exe",
"tfvc.restrictWorkspace": true,   

"team.showWelcomeMessage": true
```

I'm not able to login and `Team (i)` icon at the bottom left corner says `Unauthorized. Check your authentication credentials and try again.`

I can see pending check-ins on the Source Control tab, but when I try to check-in, I get `(team) You are signed out. Please run the 'team signin' command.`

**Comment by machintoshuser** [comment]
I am experiencing the same. Please let me know how to solve this

**Comment by diptesh-patil** [comment]
I am facing this problem too.. is there any work around for this?

**Comment by FishOfPrey** [comment]
I checked the OUTPUT window for TFVC and noticed it was trying to send the command:
tf status -noprompt -format:detailed -recursive

So I jumped over to the command prompt and tried the same thing. It came back with the status as expected.

I tried the Team: Signin command, which appeared to work.
![image](https://user-images.githubusercontent.com/2396092/53121372-fa3bfa00-35b8-11e9-90f0-a069e61a904f.png)
https://github.com/Microsoft/azure-repos-vscode/blob/master/TFVC_README.md#open-a-local-tfvc-repository-folder

That didn't seem to resolve the issue.
